### DISCUSSION TITLE
How should I call methods from other modules into my method?

### DISCUSSION THREAD

**Discussion by BorjaEst** [imk-toolkit/imk-toolkit#62] [Q&A] [answered]
How should I define a method that uses an already defined method? 
For example anomalies?

Something like this?
```py
#!/usr/bin/env python
# coding: utf-8
"""Example to use anomalies within a xr.DataArray."""
import xarray as xr
from . import anomalies


def main(dataarray):
    """Return monthly anomalies for dataarray."""
    return anomalies.main(dataarray)
```

Is there a more pythonic way you had in mind?

**Comment by ucyo** [comment, marked as answer]
There is nothing mandatory here. I personally like absolute imports, since relative imports can get messy really fast. But we have yet to define a coding standard. So. I would say go with your guts... 👍🏼  But maybe someone else has a strong opinion on this?

**Reply by BorjaEst** [reply]
I was more worried about the `main` call as it can leave to missunderstandings.
But yes, +1 regarding the absolute imports.

**Reply by ucyo** [reply]
Yes. That is true. We might wanna change that. Maybe a better idea is to rename `main` to the script name itself.

`anomalies.py` calls than the `anomalies(...)` function instead of `main(...)`. Would this be a better option?

**Reply by BorjaEst** [reply]
mm.... I think that would make testing and automation more complicated.
I would use "main" for now.
